apps/fyler/src/fyler_server.erl

```diff
@@ -384,7 +387,7 @@ handle_info({'DOWN', _, process, Pid, Reason}, #state{managers = Managers} = Sta
   case lists:keyfind(Pid, 2, maps:to_list(Managers)) of
     {Category, Pid} ->
       ?D({"manager down", Category, Reason}),
-      {noreply, State#state{managers = maps:remove(Category, Managers)}};
+      {noreply, State#state{managers = maps:put(Category, undefined, Managers)}};
```

Comment:
Line 390 is too long:       {noreply, State#state{managers = maps:put(Category, undefined, Managers)}};.